test/UniStaker.t.sol

```diff
@@ -2648,6 +2648,49 @@ contract SetAdmin is UniStakerTest {
   }
 }
 
+contract InvalidateNonce is UniStakerTest {
+  using stdStorage for StdStorage;
+
+  function testFuzz_SucessfullyInvalidatesNonceForMsgSender(address _caller, uint256 _initialNonce)
```

Comment:
nit: test name could be a little clearer from the `scopelint spec` perspective. Maybe "Increments the nonce value of the sender"